.github/workflows/compile-artifact.yml

```diff
@@ -35,7 +41,8 @@ jobs:
       - name: Configure bundler
         run: |
           mkdir -p vendor/bundle
-          bundle config path vendor/bundle
+          bundle config set --local path vendor/bundle
+          bundle config set --local gemfile Gemfile${{ inputs.gemfile_suffix }}
```

Comment:
https://github.com/rewindio/rewind-worker/runs/4508958947?check_suite_focus=true#step:4:4 It works!